cmd/depth/depth.go

```diff
@@ -58,11 +68,43 @@ func handlePkgs(t *depth.Tree, pkgs []string, outputJSON bool) error {
 		}
 
 		writePkg(os.Stdout, *t.Root, 0, false)
+		writePkgSummary(os.Stdout, *t.Root)
 	}
-
 	return nil
 }
 
+// writePkgSummary
```

Comment:
This comment can be removed or fleshed out, but as it currently is it's not doing much :)